gradle/test.gradle

```diff
@@ -167,6 +154,16 @@ subprojects {
     outputs.upToDateWhen{false}
   }
 
+  task gfshDistributedTest(type:Test) {
```

Comment:
You aren’t specifying a sourceSet so this task will inherit the test sourceSet and run only Unit tests.